RecoLocalMuon/GEMSegment/plugins/ME0SegFit.cc

```diff
@@ -343,7 +343,8 @@ ME0SegFit::SMatrixSym12 ME0SegFit::weightMatrix() {
   int row = 0;
   
   for (ME0SetOfHits::const_iterator it = hits_.begin(); it != hits_.end(); ++it) {
-    
+    if (row > 11) break; // temp due to max size of matrix
```

Comment:
it looks like there is a bug in the code, if it comes to this point.
Maybe better not to run this?